### ISSUE TITLE
A11y_ML .NET Model Builder_Object Detection-Evaluate_ScreenReader: NVDA/Narrator is not announcing 'Show undetected objects' button name when focus lands on the button.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1631]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG1.3.1; #Narrator; #NVDA; #A11yPreview; #Screen Reader; 

### Environment Details:
Microsoft Visual Studio

### Re-pro Steps:

1. Turn on Narrator/NVDA. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#) ex: WpfApp
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Object Detection" and press enter
5. Select "Azure" environment. Tab till "Setup Work Space" button and press enter and create a new experiment and tab till "next" button and press enter
6. Select input data folder and tab till "Next" button and press enter
7. Tab till "Train" button and press enter. After training is completed, navigate to evaluate tab.
8. In Evaluate tab, upload a picture in try your model section and navigate to score threshold value and increase the value unti 'Show undetected objects' gets enabled.
9. Navigate to 'Show undetected objects' button and verify whether screen announces button name or not

### Actual Result:
NVDA/Narrator is not announcing 'Show undetected objects' button when focus lands on the button.

### Expected Result:
NVDA/Narrator should announce 'Show undetected objects' button when focus lands on the button. Button name should be defined in programmatic way.

### User Impact:
Screen reader users will not know the control and might get confused if name is not announced by NVDA/Narrator.

**Comment by beccamc** [comment]
Validate on  rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by beccamc** [comment]
Here is a link the most recent Model Builder build. I’ve tested it on VS 17.0.0 Preview 5 

Model Builder: https://devdiv.visualstudio.com/_apis/resources/Containers/8681620/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix
VS: https://aka.ms/vs/17/pre/vs_Enterprise.exe

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Narrator/NVDA is reading 'Show undetected objects' button name when focus lands on the button. Hence closing the bug.

![1631_Narrator_fixed](https://user-images.githubusercontent.com/83570638/133389523-65d2a74c-0ec8-4f21-a58a-b3f1c9a3fa48.png)
![1631_NVDA_fixed](https://user-images.githubusercontent.com/83570638/133389552-08ebbc49-d47e-45f7-8dc8-031ad4c757c1.png)

**Comment by akashelijah05** [comment]
#Closed;
